tests/parametric/test_span_links.py

```diff
@@ -1,33 +1,41 @@
 import json
+import pytest
 
-from ddapm_test_agent.trace import Span
-from ddapm_test_agent.trace import root_span
-from utils.parametric.spec.trace import SAMPLING_PRIORITY_KEY, ORIGIN
-from utils.parametric.spec.trace import find_span
-from utils.parametric.spec.trace import find_trace_by_root
+from utils.parametric.spec.trace import ORIGIN
 from utils.parametric.spec.trace import span_has_no_parent
-from utils.parametric.test_agent import get_span
-from utils import context, scenarios, missing_feature, bug
+from utils import scenarios
 from utils.parametric._library_client import Link
 
 
 @scenarios.parametric
 class Test_Span_Links:
     @staticmethod
     def _get_span_links(span):
-        if context.library >= "python@2.4.0":
-            # trace API v0.5
-            encoded_span_links = span.get("meta", {}).get("_dd.span_links")
-            if not encoded_span_links:
-                return None
-
-            return json.loads(encoded_span_links)
-
-        # trace API v0.4
-        return span.get("span_links")
-
-    @bug(library="python", reason="Since using v0.5 API, trace_id are differents")
-    def test_span_started_with_link(self, test_agent, test_library):
```

Comment:
`test_span_started_with_link` is replaced by `test_span_started_with_link_v04` and `test_span_started_with_link_v05`. Here we are testing both trace api versions correctly serialize span links